test/fixtures/test_terraform_variables/test.tf

```diff
@@ -15,7 +15,7 @@ resource "test" "test1" {
   test_map        = var.map2
   test_bool       = var.test1
   test_list       = var.test2
-  test_neted_map  = var.map2[var.map1["map1key1"]]]
+  test_neted_map  = var.map2[var.map1["map1key1"]]
```

Comment:
this and t`test_without_variables_block.tf` were making vscode show an error...so I went ahead and fixed them to not have to see red. :)

Reply:
I kindly ask you to revert the fix in that `test.tf` file. It is used for testing purposes 😅
The fix on that file is causing this [test](https://github.com/Checkmarx/kics/blob/master/pkg/parser/terraform/terraform_test.go#L238) to fail.

Reply:
eheh this is what I get for not running the *full test suite before committing.